client/landing/jetpack-cloud/components/daily-backup-status/index.jsx

```diff
@@ -135,34 +143,131 @@ class DailyBackupStatus extends Component {
 						{ translate( 'Contact support' ) }
 					</Button>
 				</div>
-			</Card>
+			</>
 		);
 	}
 
-	renderNoBackups() {
-		const { translate } = this.props;
+	renderNoBackup() {
+		const { translate, selectedDate, onDateChange } = this.props;
 
-		// todo: remove the mock dates by the real dates
-		const lastBackupAt = 'Yesterday 16:02';
-		const nextBackupAt = 'Today 16:02';
+		const displayDate = selectedDate.format( 'll' );
+		const nextDate = selectedDate.clone().add( 1, 'days' );
+		const displayNextDate = nextDate.format( 'll' );
 
 		return (
-			<Fragment>
-				<Gridicon icon="sync" />
+			<>
+				<Gridicon icon="cloud-upload" className="daily-backup-status__gridicon-no-backup" />
+				<div className="daily-backup-status__title">{ translate( 'No backup' ) }</div>
```

Comment:
:information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **21** times:
`translate( 'No backups' )` *ES Score: 7*